accumulator/pollardproof.go

```diff
@@ -37,8 +34,7 @@ func (p *Pollard) IngestBatchProof(bp BatchProof) error {
 			if node.niece[lr] == nil {
 				node.niece[lr] = new(polNode)
 				node.niece[lr].data = proofMap[pos]
-				// fmt.Printf("------wrote %x at %d\n", proofMap[pos], pos)
-				if node.niece[lr].data == empty {
+				if node.niece[lr].data == (Hash{}) {
```

Comment:
would this not allocate 32 bytes for every comparison? or is the go compiler smart enough to avoid that?

Reply:
Looks like the compiler is not smart enough to do so. I'll revert this